index.html

```diff
@@ -31,90 +33,98 @@
-             <p>Не только китайского, но и местного производства!</p>
+           <li class="features-list-item">
+             <h3 class="features-list-title">Сувениры</h3>
+             <p class="features-list-text">Не только китайского, но и местного производства!</p>
            </li>
          </ul>
        </li>
-       <li>
-        <h2>Там есть<br> Мост дьявола</h2>
-        <p>— №2 —</p>
-        <p>Да, по нему можно пройти! Если конечно вы осмелитесь</p>
+       <li class="features-item">
+        <div class="features-desc">
+          <h2 class="features-title">Там есть<br> Мост дьявола</h2>
+          <p class="features-text">— №2 —</p>
+          <p class="features-text">Да, по нему можно пройти! Если конечно вы осмелитесь</p>
+        </div>
       </li>
-      <li>
-        <h3>небольшая площадь</h3>
-        <p>— №3 —</p>
-        <p>все достопримечательности находятся очень близко</p>
+      <li class="features-item">
+        <h3 class="features-title">небольшая площадь</h3>
+        <p class="features-text">— №3 —</p>
+        <p class="features-text">все достопримечательности находятся очень близко</p>
       </li>
-      <li>
-        <h3>красивая дорога</h3>
-        <p>— №4 —</p>
-        <p>ехать в седону из лас-вегаса совсем  не скучно!</p>
+      <li class="features-item">
+        <h3 class="features-title">красивая дорога</h3>
+        <p class="features-text">— №4 —</p>
+        <p class="features-text">ехать в седону из лас-вегаса совсем  не скучно!</p>
       </li>
-      <li>
-        <h3>мало туристов</h3>
-        <p>— №5 —</p>
-        <p>большинство едет в гранд каньон и толпится там</p>
+      <li class="features-item">
+        <h3 class="features-title">мало туристов</h3>
+        <p class="features-text">— №5 —</p>
+        <p class="features-text">большинство едет в гранд каньон и толпится там</p>
       </li>
     </ul>
   </section>
-  <section class="form">
-    <div class="form__title">
-      <h2>Заинтересовались?</h2>
-      <p class="">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в седоне</p>
+  <section class="main-form">
+    <div class="main-form-top">
+      <h2 class="main-form-title">Заинтересовались?</h2>
+      <p class="main-form-desc">Укажите предполагаемые даты поездки,<br> и мы покажем вам лучшие предложения гостиниц в седоне</p>
     </div>
-    <div class="form__bottom">
-      <button type="button">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
+    <div class="main-form-bottom">
+      <button type="button" class="main-form-button">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
     </div>
   </section>
-  <section class="map">
+  <section class="main-map">
     <iframe src="" frameborder="0"></iframe>
   </section>
 </main>
-<footer>
-  <div>
-    <a href="">#visitSEDONA</a>
-    ТЕЛ: <a href="tel:+74956660166">+7(495) 666-01-66</a>
+<footer class="main-footer container">
+  <div class="main-footer-contact">
+    <a href="/" class='main-footer-link'>#visitSEDONA</a>
```

Comment:
https://up.htmlacademy.ru/htmlcss/30/criteries#b15
Используй какой-то один тип кавычек во всем html-файле
```suggestion
    <a href="/" class="main-footer-link">#visitSEDONA</a>
```